draft-ietf-rats-corim.md

```diff
@@ -2087,7 +2088,7 @@ The handling of dynamic Evidence transformation algorithms is out of scope for t
 {: rtt2-enum}
 * For each e in RVT.`ref-claims`:
 
-> > **copy**(e.`measurement-map`, `rv`.`addition`.`element-list`.`element-map`)
+> > **copy**(e.`measurement-map`, `rv`.`condition`.`element-list`.`element-map`)
```

Comment:
Isn't it both addition and condition for `rv`?

Reply:
I deliberately omitted the `addition`, as upon matching `evidence` ECTs the claims from `evidence` gets moved to `addition` section of the RVP ECTs. So double movement risks adding confusion to the reader!